pkgs/build-support/gradle/fetch-deps.nix

```
@@ -0,0 +1,32 @@
+{ mitm-cache
+, lib
+}:
+
+{ data, pname ? throw "Please pass pname to fetchDeps", name ? "${pname}-deps", ... } @ attrs:
```

```suggestion
{ data, pname, name ? "${pname}-deps", ... } @ attrs:
```
nix should already complain if no pname is passed in here IIRC

---

pname is meant to be optional here, if pname isn't passes, name must be set instead